src/public/client.ts

```diff
@@ -76,6 +112,11 @@ create.onclick = (): void => {
       'Content-Type': 'application/json'
     }
   }).then(() => {
-    window.location.reload()
+    render()
   })
+  port.value = ''
+  ipAddress.value = ''
+  subDomain.value = ''
 }
+
+render()
```

Comment:
if you are calling render, then you can remove your `fetch('/api/mappings'...)` that you did outside of `render`